### PR TITLE
Release changes 

### PR DESCRIPTION
**Pull request by kanika1894**
It includes

1. ReadMe(documentation) updates
2. Fix boolean checks on multiple parameters
3. Fix logic to read acr from image name

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,6 +134,18 @@ jobs:
     location: 'east us'
 ```
 
+# Local Development and Testing
+
+If you wish to develop and test changes against a local fork or development repo, you can do so by including the `node_modules` in tagged release branch. Note that the `aci-deploy` repository does not include these modules in the master branch, so you cannot point your action to `aci-deploy/master` to pick up recent commits.
```
Comment (kaverma) [inline README.md:139]: I think these are generic and we can also include in Azure/Actions guidelines if it is not present

**Comment by kaverma** [comment]
Is this a cherry-pick? And have we run the validations from azure-integration-tests?
Was also thinking if we can automate the running of tests in PR checks. Jyotsna did it for some Actions, we could check with her

**Review by kaverma — COMMENTED** [review]
Had few questions in the comments

File: README.md (hunk 1 above)
Comment (BALAGA-GAYATRI) [reply README.md:139]: @kanika1894 you can refer to this link here directly for [developer-guildelines](https://github.com/Azure/actions/blob/main/docs/developer-guildelines.md).

Comment (kanika1894) [reply README.md:139]: @BALAGA-GAYATRI as discussed, are the guidelines updated to focus on the node_modules part?

**Pull request merged**